stl/inc/numbers

```diff
@@ -0,0 +1,193 @@
+// numbers standard header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _NUMBERS_
+#define _NUMBERS_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#if !_HAS_CXX20
+#pragma message("The contents of <numbers> are available only with C++20 or later.")
+#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
+#ifdef __cpp_lib_concepts
+#include <concepts>
+#endif // ^^^ defined(__cpp_lib_concepts) ^^^
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace numbers {
+    template <class _Ty>
+    struct _Invalid {
+        static_assert(_Always_false<_Ty>, "A program that instantiates a primary template of a mathematical constant "

```

Comment:
`_Always_false` is defined by `<xstddef>`. That's dragged in by `<concepts>`, but in non-concepts mode, it won't be dragged in.

Reply:
I'll push a change to resolve this.

Reply:
🤦‍♂ Forgot to remove other headers in my test project.

Reply:
No worries, it's an easy thing to miss. Fortunately we have an "include each header alone" test case (although it doesn't run every compiler configuration), so when our tests are online, this minor mistake category will be harder to make.